src/Sylius/Bundle/AdminBundle/Resources/views/Currency/_form.html.twig

```diff
@@ -0,0 +1,10 @@
+{{ form_errors(form) }}
+<div class="one field">
+    <div class="required field">
+        <label class="required" for="sylius_currency_code"> Name </label>
```

Comment:
I had to change AddCodeFormSubscirber to add this translation, which is not use in any other place. That is better solution for me, because it not required adapting work of subscriber to this little change.